canonical/ubuntu.com
# WD-3904-credentials-schedule

## Done

- [List of work items including drive-bys - remember to add the why and what of this work.]

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- visit /credentials/schedule?contractItemID=<contractItemID>

## Issue / Card

Fixes # [WD-3904](https://warthogs.atlassian.net/browse/WD-3904)

## Screenshots

![image](https://github.com/canonical/ubuntu.com/assets/30973042/4f504895-b2fc-485f-a927-ca1203c7c8e2)


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-3904]: https://warthogs.atlassian.net/browse/WD-3904?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
